third_party/2/pycurl.pyi

```diff
@@ -45,37 +45,339 @@ class CurlShare(object):
-NOSIGNAL = ...  # type: int
-POST = ...  # type: int
-POSTFIELDS = ...  # type: int
-POSTFIELDSIZE = ...  # type: int
-PRIMARY_IP = ...  # type: int
-PROGRESSFUNCTION = ...  # type: int
-PROXY = ...  # type: int
-READFUNCTION = ...  # type: int
-RESPONSE_CODE = ...  # type: int
-SSLCERT = ...  # type: int
-SSLCERTPASSWD = ...  # type: int
-SSLKEY = ...  # type: int
-SSLKEYPASSWD = ...  # type: int
-SSL_VERIFYHOST = ...  # type: int
-SSL_VERIFYPEER = ...  # type: int
-TIMEOUT_MS = ...  # type: int
-UPLOAD = ...  # type: int
-URL = ...  # type: int
-WRITEFUNCTION = ...  # type: int
+ADDRESS_SCOPE = ... # type: int
+APPCONNECT_TIME = ... # type: int
+AUTOREFERER = ... # type: int
+BUFFERSIZE = ... # type: int
+CAINFO = ... # type: int
+CAPATH = ... # type: int
+COMPILE_LIBCURL_VERSION_NUM = ... # type: int
+COMPILE_PY_VERSION_HEX = ... # type: int
+CONNECTTIMEOUT = ... # type: int
+CONNECTTIMEOUT_MS = ... # type: int
+CONNECT_ONLY = ... # type: int
+CONNECT_TIME = ... # type: int
+CONTENT_LENGTH_DOWNLOAD = ... # type: int
+CONTENT_LENGTH_UPLOAD = ... # type: int
+CONTENT_TYPE = ... # type: int
+COOKIE = ... # type: int
+COOKIEFILE = ... # type: int
+COOKIEJAR = ... # type: int
+COOKIELIST = ... # type: int
+COPYPOSTFIELDS = ... # type: int
+CRLF = ... # type: int
+CRLFILE = ... # type: int
+CSELECT_ERR = ... # type: int
+CSELECT_IN = ... # type: int
+CSELECT_OUT = ... # type: int
+CURL_HTTP_VERSION_1_0 = ... # type: int
+CURL_HTTP_VERSION_1_1 = ... # type: int
+CURL_HTTP_VERSION_LAST = ... # type: int
+CURL_HTTP_VERSION_NONE = ... # type: int
+CUSTOMREQUEST = ... # type: int
+DEBUGFUNCTION = ... # type: int
+DNS_CACHE_TIMEOUT = ... # type: int
+DNS_USE_GLOBAL_CACHE = ... # type: int
+EFFECTIVE_URL = ... # type: int
+EGDSOCKET = ... # type: int
+ENCODING = ... # type: int
+FAILONERROR = ... # type: int
+FILE = ... # type: int
+FOLLOWLOCATION = ... # type: int
+FORBID_REUSE = ... # type: int
+FORM_CONTENTS = ... # type: int
+FORM_CONTENTTYPE = ... # type: int
+FORM_FILE = ... # type: int
+FORM_FILENAME = ... # type: int
+FRESH_CONNECT = ... # type: int
+FTPAPPEND = ... # type: int
+FTPAUTH_DEFAULT = ... # type: int
+FTPAUTH_SSL = ... # type: int
+FTPAUTH_TLS = ... # type: int
+FTPLISTONLY = ... # type: int
+FTPMETHOD_DEFAULT = ... # type: int
+FTPMETHOD_MULTICWD = ... # type: int
+FTPMETHOD_NOCWD = ... # type: int
+FTPMETHOD_SINGLECWD = ... # type: int
+FTPPORT = ... # type: int
+FTPSSLAUTH = ... # type: int
+FTPSSL_ALL = ... # type: int
+FTPSSL_CONTROL = ... # type: int
+FTPSSL_NONE = ... # type: int
+FTPSSL_TRY = ... # type: int
+FTP_ACCOUNT = ... # type: int
+FTP_ALTERNATIVE_TO_USER = ... # type: int
+FTP_CREATE_MISSING_DIRS = ... # type: int
+FTP_ENTRY_PATH = ... # type: int
+FTP_FILEMETHOD = ... # type: int
+FTP_RESPONSE_TIMEOUT = ... # type: int
+FTP_SKIP_PASV_IP = ... # type: int
+FTP_SSL = ... # type: int
+FTP_SSL_CCC = ... # type: int
+FTP_USE_EPRT = ... # type: int
+FTP_USE_EPSV = ... # type: int
+GLOBAL_ALL = ... # type: int
```

Comment:
This is redefined from line 7. The tests complain about a few more. Make sure to ./runtests.sh locally.